src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -197,6 +520,12 @@ describe('PaypalCommerceButtonStrategy', () => {
         expect(paypalCommercePaymentProcessor.initialize).toHaveBeenCalledWith(obj);
     });
 
+    it('no messages', async () => {
```

Comment:
It might be more understandable to update the test message to:

```suggestion
    it('does not render any paypal message', async () => {
```

Reply:
fixed